.github/workflows/website.yml

```diff
@@ -0,0 +1,30 @@
+name: Website
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'website/**'
+
+jobs:
+  publish:
+    if: ${{ github.ref == 'refs/heads/master' }}
+    name: Publish to GitHub Pages
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node from node version file
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
+      - name: Install dependencies
+        run: yarn --frozen-lockfile --non-interactive --silent --ignore-scripts
+      - name: Build website
+        run: yarn workspace @loki/website build
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
It seems we don't have to worry about generating the `GITHUB_TOKEN` according to this 👇 


> For newbies of GitHub Actions: Note that the GITHUB_TOKEN is NOT a personal access token. A GitHub Actions runner automatically creates a GITHUB_TOKEN secret to authenticate in your workflow. So, you can start to deploy immediately without any configuration.

🔗 https://github.com/marketplace/actions/github-pages-action